schema/crdb/dbinit.sql

```diff
@@ -5556,6 +5556,35 @@ CREATE TABLE IF NOT EXISTS omicron.public.db_metadata (
     -- (Optional) Semver representation of the DB version to which we're upgrading
     target_version STRING(64),
 
+    -- During the nexus boot process, it must check this table to understand the
```

Comment:
I was going through this carefully and came up with [this](https://docs.google.com/document/d/1jbzwCkBSt8plJrK11j-nsWFYgxJIolmptrylmym3KTA/edit?tab=t.0):
<img width="1028" height="260" alt="Screenshot 2025-08-01 at 10 12 09 AM" src="https://github.com/user-attachments/assets/9400ad1f-376d-4a8d-8e0e-86314c9719ea" />
Does that seem right?

Reply:
Yeah, that does seem accurate - and that was my intent through implementation. Hopefully the tests in `nexus/db-queries/src/db/datastore/db_metadata.rs` support this too!